flowey/flowey_hvlite/src/pipelines/checkin_gates.rs

```diff
@@ -627,8 +627,10 @@ impl IntoPipeline for CheckinGatesCli {
 
             let (pub_openhcl_igvm, use_openhcl_igvm) =
                 pipeline.new_artifact(format!("{arch_tag}-openhcl-igvm"));
-            let (pub_openhcl_igvm_extras, _use_openhcl_igvm_extras) =
+            let (pub_openhcl_igvm_extras_dev, _use_openhcl_igvm_extras_dev) =
                 pipeline.new_artifact(format!("{arch_tag}-openhcl-igvm-extras"));
+            let (pub_openhcl_baseline, _use_openhcl_baseline) =
```

Comment:
this should only be published / built in CI configurations. otherwise, you're doing duplicate work with the check size task